README.md

```diff
@@ -58,14 +61,23 @@ To install and build static assets:
 
 ``` $ npm run dev  # watch asset files and recompile when changed ```
 
-Then, to build and run the go binary
+Then, to set up the database and run PostgREST, which our Go code uses for persistence:
+
+``` $ cd db/ ```
+
+``` $ sudo su postgres -c 'bash init_sql.sh' ```
```

Comment:
Hmm okay, good to know. The key is becoming the `postgres` user then running `bash init_sql.sh`.